src/sonic-frr/patch/0011-Changes-to-send-ipv6-link-local-address-as-nexthop-to-fpmsyncd.patch

```diff
@@ -0,0 +1,50 @@
+From 6a74cc3b97f405473c7500d5cd0adadba854067f Mon Sep 17 00:00:00 2001
```

Comment:
Do we need still need this patch file, looks like we are not using it

Reply:
As FRR updated to 7.5 version, not needed this patch file at all.